README.md

```diff
@@ -46,6 +46,49 @@ Extend @ni/eslint-config/typescript in the ESLint configuration. Configure the @
 }
 ```
 
+### Angular
+
+ESLint support for Angular is provided by [@angular-eslint](https://github.com/angular-eslint/angular-eslint#readme). It's recommended to use @angular-eslint/schematics to
+configure ESLint for Angular projects especially when migrating from TSLint.
+
+#### Generating a Workspace
+
+Workspaces [generated with @angular-eslint](https://github.com/angular-eslint/angular-eslint#quick-start-with-angular-and-eslint) will generate projects with @angular-eslint.
+
+```bash
+> npm i -g @angular/cli @angular-devkit/{core,schematics} @angular-eslint/schematics
```

Comment:
This syntax does not work on windows clis. Can you use a command that is cross-platform?
```
npm i -g @angular/cli @angular-devkit/core @angular-devkit/schematics @angular-eslint/schematics
```

Reply:
Thanks for pointing this out. I only ran it on macOS.